### PR TITLE
feat(chart): add dependency package

### PR DESCRIPTION
**Pull request by cndoit18**
Signed-off-by: cndoit18 <cndoit18@outlook.com>

### CODE REVIEW

**Comment by thcdrt** [comment]
Since 2.3.0 kubebuilder publish binary instead of archive.

**Review by bitsf — APPROVED** [review]

File: config/helm/deployment/patch_deployment_config.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
       - name: manager
         env:
         - name: POSTGRES_SECRETS_TEMPLATE
-          value: '{{.Values.postgresOperator.configKubernetes.secret_name_template|quote}}'
+          value: '{{index .Values "postgres-operator" "configKubernetes" "secret_name_template"|quote}}'
```
Comment (cndoit18) [inline config/helm/deployment/patch_deployment_config.yaml:12]: see https://github.com/helm/helm/issues/2192

File: charts/harbor-operator/values.yaml [hunk 2]
```diff
@@ -252,3 +248,20 @@ tolerations: []
...
+minio-operator:
+  # minio-operator.enabled -- Whether to enabled [MinIO Operator](https://github.com/minio/operator)
+  enabled: false
+
+postgres-operator:
+  # postgres-operator.enabled -- Whether to enabled [Postgres operator](https://github.com/zalando/postgres-operator)
+  enabled: false
+
+  configKubernetes:
+    secret_name_template: "{username}.{cluster}.credentials"
```
Comment (cndoit18) [inline charts/harbor-operator/values.yaml:267]: This will allow us to use the same `secret_name_template` as the `postgres-operator`

**Review by Ghostbaby — APPROVED** [review]
LGTM

**Pull request merged**